src/metrics/rules/validator/options.go

```diff
@@ -39,6 +40,8 @@ const (
 
 	// By default we allow at most one level of rollup in a pipeline.
 	defaultMaxRollupLevels = 1
+
+	tagTimerType = "timertype"
```

Comment:
Just confirming it's `timertype` and not `timerType`, right?

Reply:
Yup

Reply:
Ideally this can be fed from config files, right now `timertype` is not known to the system, and users have the ability to config the tag name for aggregation types it does not have to be `timertype`, we even once considered to use `aggType` so it can be applied to all metric types.

Reply:
Will change this to be config driven